automation/dependDockerfile/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM quay.io/openshift/origin-jenkins-agent-base:v4.0.0
+# Jenkins slave requirements
+RUN yum makecache \
+ && yum erase -y java-1.8.0-openjdk-headless \
+ && yum -y install openssl java-1.8.0-openjdk \
+ && pushd /etc/pki/ca-trust/source/anchors \
+ && curl -s -k -O https://password.corp.redhat.com/RH-IT-Root-CA.crt \
```

Comment:
@lyrch what do you of this? considering its going upstream.

Reply:
I'm not sure. I don't like exposing details like that, but it is also available outside the firewall.

Reply:
I have the same mix feelings about this, but its a [known "secret"](
https://www.google.com/search?hl=en&ei=r4OHXIG3AeqZjwTF04K4CA&q=%22password.corp.redhat.com%22&oq=%22password.corp.redhat.com%22&gs_l=psy-ab.3...4390.4787..5607...0.0..0.227.441.2-2......0....1..gws-wiz.1wdvFlltK6s) and its not harmful in anyway I can think of.

But from upstream perspective this should be optional. @shay6 I think you can add the project to Quay.io and then you won't need to update docker images downstream manually.

Reply:
I need more explanation about this process

Reply:
BTW, I've pushed it to our Registry

Reply:
https://quay.io/tutorial/

Reply:
You can also install this internal Red Hat cert by rpm.

Reply:
which rpm?

Reply:
I'm fine with how it is